tests/api/io/test_read.py

```diff
@@ -32,58 +32,74 @@ def test_read_tsv(self):
                 [16, 27, 190.32, True],
             ]
         )
-        assert ann_data.X.shape == (5, 4)
-        assert (ann_data.X == matrix).all()
-        assert ann_data.var_names.to_list() == ["patient_id", "age", "height", "gamer"]
-        assert (ann_data.layers["original"] == matrix).all()
-        assert id(ann_data.layers["original"]) != id(ann_data.X)
+        assert adata.X.shape == (5, 4)
+        assert (adata.X == matrix).all()
+        assert adata.var_names.to_list() == ["patient_id", "age", "height", "gamer"]
+        assert (adata.layers["original"] == matrix).all()
+        assert id(adata.layers["original"]) != id(adata.X)
 
     def test_read_csv_without_index_column(self):
-        ann_data = DataReader.read(dataset_path=f"{_TEST_PATH}/dataset3.csv")
+        adata = DataReader.read(dataset_path=f"{_TEST_PATH}/dataset3.csv")
         matrix = np.array(
             [[1, 14, 500, False], [2, 7, 330, False], [3, 10, 800, True], [4, 11, 765, True], [5, 3, 800, True]]
         )
-        assert ann_data.X.shape == (5, 4)
-        assert (ann_data.X == matrix).all()
-        assert ann_data.var_names.to_list() == ["clinic_id", "los_days", "b12_values", "survival"]
-        assert (ann_data.layers["original"] == matrix).all()
-        assert id(ann_data.layers["original"]) != id(ann_data.X)
-        assert list(ann_data.obs.index) == ["0", "1", "2", "3", "4"]
+        assert adata.X.shape == (5, 4)
+        assert (adata.X == matrix).all()
+        assert adata.var_names.to_list() == ["clinic_id", "los_days", "b12_values", "survival"]
+        assert (adata.layers["original"] == matrix).all()
+        assert id(adata.layers["original"]) != id(adata.X)
+        assert list(adata.obs.index) == ["0", "1", "2", "3", "4"]
+
+    def test_read_pdf(self):
```

Comment:
Can we please add one test for a PDF with a proper header and one where the top left is empty?